contracts/src/ten_erc20/TENToken.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+// Compatible with OpenZeppelin Contracts ^5.0.0
+pragma solidity ^0.8.22;
+
+import {ERC20Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import {ERC20PausableUpgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20PausableUpgradeable.sol";
+import {ERC20PermitUpgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20PermitUpgradeable.sol";
+import {ERC20VotesUpgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20VotesUpgradeable.sol";
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {NoncesUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/NoncesUpgradeable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+/// @custom:security-contact security@ten.xyz
+contract TENToken is Initializable, ERC20Upgradeable, ERC20PausableUpgradeable, OwnableUpgradeable, ERC20PermitUpgradeable, ERC20VotesUpgradeable {
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor() {
+        _disableInitializers();
+    }
+
+    function initialize(address recipient, address initialOwner)
+        public initializer
+    {
+        __ERC20_init("TENToken", "TEN");
```

Comment:
When metadata is pulled by front ends if they dispaly name instead of symbol it will show up as TENToken

Reply:
it should be "TEN" or "TEN Token" ?

Reply:
GPT says it's usually different, examples:

	•	Uniswap:
"Uniswap", "UNI"
	•	Aave:
"Aave Token", "AAVE"
	•	Maker:
"Dai Stablecoin", "DAI"
	•	USDC:
"USD Coin", "USDC"

I think "TEN Token" with space